src/components/shared/Modal.js

```diff
@@ -30,7 +35,11 @@ export class Modal extends Component {
 
     return (
       <div className="modal-wrapper" onClick={this.props.handleClose}>
-        <div className={classnames("modal", status)} onClick={this.onClick}>
+        <div
```

Comment:
perhaps we want to be able to pass in a modal class name that can override the height? `className={classnames("modal", modalClass)}`